src/new-components/button-link/button-link.tsx

```diff
@@ -14,6 +14,7 @@ const ButtonLink = forwardRefWithAs<ButtonLinkProps>(function ButtonLink(
     { component = 'a', variant, size = 'default', openInNewTab = false, ...props },
     ref,
 ) {
+    const { className } = props
```

Comment:
Could be extracted right in the function signature:
```suggestion
    { component = 'a', variant, size = 'default', openInNewTab = false, className, ...props },
    ref,
) {
```

_not sure if this passes prettier_

Reply:
Yep, you're right 🤦🏻‍♂️ Have updated this now.